pkg/upgrade/control_plane.go

```diff
@@ -681,6 +688,24 @@ func (u *ControlPlaneUpgrader) updateAndUploadKubeadmKubernetesVersion() error {
 	return nil
 }
 
+func (u *ControlPlaneUpgrader) removeNodeFromKubeadmConfigMap(nodeName string) error {
+	original, err := u.targetKubernetesClient.CoreV1().ConfigMaps("kube-system").Get("kubeadm-config", metav1.GetOptions{})
```

Comment:
ideally `metav1.NamespaceSystem` should be used instead of the literal.
`kubeadm-config` should be constant too.

Reply:
i globally swapped this in (in this file), FYI